src/test/java/net/smartcosmos/cluster/userdetails/AbstractTestResource.java

```diff
@@ -0,0 +1,67 @@
+package net.smartcosmos.cluster.userdetails;
```

Comment:
I agree. Recently, I got used to using something like this:
```
// region Setup

@Before
public void setUp() {
    ....
}

@After
public void tearDown() {
    ....
}

// endregion
```

Like this, everything is in the test class and can easily be adapted to the particular class that is tested, but since the block can be collapsed, it doesn't at disturbing noise.